README.md

```diff
@@ -1,50 +1,50 @@
-Backwards compatibility with the vkd3d standalone API is not a goal of this project.
+Efficiency and compatibility are important goals at the expense of compatibility with older drivers and systems.
+Modern Vulcan extensions and features are being used aggressively to improve performance and compatibility.
+For the best experience, it is recommended to use the latest drivers that you get your hands on.
+Compatibility with the vkd3d standalone API is not the goal of this project.
 
 ------
 
 ## Cloning the repo
 
-To clone the repo you should run:
+To clone a turnip, run:
 ```
 git clone --recursive https://github.com/HansKristian-Work/vkd3d-proton
 ```
-in order to pull in all the submodules which are needed for building.
+to pull all the submodules needed for construction.
 
 ## Building VKD3D
 
 ### Requirements:
 - [wine](https://www.winehq.org/) (for `widl`) [for native builds]
-  - On Windows this may be substituted for [Strawberry Perl](http://strawberryperl.com/) as it ships `widl` and is easy to find and install -- although this dependency may be eliminated in the future.
+  - On Windows this may be substituted for [Strawberry Perl](http://strawberryperl.com/) as it sends a `widl` and is easy to find and install - although this dependency can be fixed in the future.
 - [Meson](http://mesonbuild.com/) build system (at least version 0.51)
-- [glslang](https://github.com/KhronosGroup/glslang) compiler
+- [glslang](https://github.com/KhronosGroup/glslang) translator
 - [Mingw-w64](http://mingw-w64.org/) compiler, headers and tools (at least version 7.0) [for cross-builds for d3d12.dll which are default]
 
 ### Building:
 #### The simple way
-Inside the VKD3D directory, run:
+Within the VKD3D directory, run:
 ```
 ./package-release.sh master /your/target/directory --no-package
 ```
 
-This will create a folder `vkd3d-master` in `/your/target/directory`, which contains both 32-bit and 64-bit versions of VKD3D, which can be set up in the same way as the release versions as noted above.
+This will create a `vkd3d-master` folder in `/your/target/directory`, which contains 32-bit and 64-bit versions of VKD3D, which you can set up in the same way as the release versions as mentioned above.
 
-If you want to build natively (ie. for `libvkd3d.so`), pass `--native` to the build script. This option will make it build using your system's compilers.
+To build the original (eg for `libvkd3d.so`), enter` --native` in the build script. With this option, it will be built using your system's compilers.
 
-In order to preserve the build directories for development, pass `--dev-build` to the script. This option implies `--no-package`. After making changes to the source code, you can then do the following to rebuild VKD3D:
+To save build directories for development, download `--dev-build` in the script. This option means "--no-package". After changing the source code, you can do the following to restore VKD3D:
```

Comment:
You are not downloading `--dev-build`; you are passing that argument (telling it explicitly to preserve the build directories).
Restoring and rebuilding are very different.